tensorflow/core/kernels/mkl_fused_batch_norm_op.cc

```diff
@@ -818,10 +829,29 @@ class MklFusedBatchNormOp : public OpKernel {
     // set NAN variance value in case of empty input tensor
     auto saved_variance_data = (*saved_variance_tensor)->flat<U>().data();
     std::fill_n(saved_variance_data, num_elements, static_cast<U>(NAN));
+
+    // Changes to support reserved_space_3 parameter in FusedBatchNormV3.
+    // TODO: This parameter functionality is not implemented on CPU.
+    //       It is used to hold intermediate results. So the allocated
+    //       memory is filled with NANs.
+    if (reserved_space) {
+      DCHECK(reserved_space_tensor != nullptr);
+
+      MklDnnShape mkl_shape_reserved_space;
+      mkl_shape_reserved_space.SetMklTensor(false);
+      AllocateOutputSetMklShape(context, kReservedSpaceIndex,
+                                reserved_space_tensor, tf_shape_scale,
+                                mkl_shape_reserved_space);
+      DCHECK((*reserved_space_tensor) != nullptr);
+      // set NAN variance value in case of empty input tensor
```

Comment:
This is not variance value, please correct the description. 
Nit: Please also capitalize the first letter of the sentence.

Reply:
Done